OlinkAnalyze/R/dist_plot.R

```diff
@@ -58,21 +58,13 @@ olink_dist_plot <- function(df, color_g = 'QC_Warning', ...) {
     ggplot2::scale_x_discrete(labels = function(x) gsub(reg, "", x), ...)
   }
 
-  # Remove "Olink", Target 96" and "Target 48" from the panel name(s)
+  #If not all are Pass, the QC_Warning is set as warning for plotting purposes
   df_OlinkID_fixed <- df_OlinkID %>%
-    dplyr::mutate(Panel = Panel  %>% stringr::str_replace("Olink ", "") %>%
-             stringr::str_replace("Target 96 ", "") %>%
-             stringr::str_replace("Target 48 ", "")) %>%
+    dplyr::mutate(Panel = Panel %>% stringr::str_replace("Olink ", "")) %>%
     dplyr::group_by(SampleID, Index, Panel) %>%
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "WARN",
-                                       "Warning",
-                                       QC_Warning)) %>%
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "PASS",
-                                       "Pass",
-                                       QC_Warning)) %>%
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "FAIL",
-                                       "Fail",
-                                       QC_Warning)) %>%
+    dplyr::mutate(QC_Warning = dplyr::if_else(all(toupper(QC_Warning) == 'PASS'),
```

Comment:
To me aesthetically it looks better with Warning vs WARN

Code wise both work equally well i'd say...

Reply:
I agree, I was thinking about this as well, but then didn't loop back.